open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -62,46 +62,49 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    print(factors(12)) # Should output [1, 2, 3, 4, 6, 12]
</pre><p>Let's change the print to be <code>assert factors(12) == [1, 2, 3, 4, 6, 12]</code></p><p> </p><p>And, please add an empty line before the assert.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>